CP01CNSM/CP01CNSM_R00011_ingest.csv

```diff
@@ -10,20 +10,20 @@ mi.dataset.driver.velpt_ab.velpt_ab_recovered_driver,/omc_data/whoi/OMC/CP01CNSM
 mi.dataset.driver.pco2w_abc.dcl.pco2w_abc_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/cg_data/dcl36/pco2w/*.pco2w.log,CP01CNSM-MFD35-05-PCO2WB000,recovered_host,Expected,
 mi.dataset.driver.pco2w_abc.pco2w_abc_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/instruments/dcl36/PCO2W*/SAMI*.txt,CP01CNSM-MFD35-05-PCO2WB000,recovered_inst,Expected,
 mi.dataset.driver.phsen_abcdef.dcl.phsen_abcdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/cg_data/dcl36/phsen2/*.phsen2.log,CP01CNSM-MFD35-06-PHSEND000,recovered_host,Expected,
-mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/instruments/dcl36/PHSEN*/SAMI_P0099_*.txt ,CP01CNSM-MFD35-06-PHSEND000,recovered_inst,Expected,
+mi.dataset.driver.phsen_abcdef.phsen_abcdef_recovered_driver,/omc_data/whoi/OMC/CP01CNSM/R00011/instruments/dcl36/PHSEN*/SAMI_*.txt,CP01CNSM-MFD35-06-PHSEND000,recovered_inst,Expected,

```

Comment:
Filemask was not finding data, I removed the space at the end and made it more generic